### PR TITLE
Log level flag for http requests

### PR DESCRIPTION
**Pull request by rtwent**
https://lead-in.monday.com/boards/72721864/pulses/5154589834

### CODE REVIEW

File: src/HttpHelper/HttpClientHelper/HttpClient.php [hunk 1]
```diff
@@ -103,7 +128,9 @@ private function request(
                 $iHttpCode = $e->getCode();
             }
 
-            Logger::error("$sAction : error while requesting $sMethod $sEffectiveUri - $iHttpCode", $logContext, \array_merge($aLogData, [
+            $sLogLevel = $bErrorLogLevelForHttpRequest ? 'error' : 'notice';
+
+            Logger::$sLogLevel("$sAction : error while requesting $sMethod $sEffectiveUri - $iHttpCode", $logContext, \array_merge($aLogData, [
```
Comment (pbernad-leadin) [inline src/HttpHelper/HttpClientHelper/HttpClient.php:133]: need to disable logs not change the level, we don't want to have `notice` for all error responses but disable default logs and handle them manually in the client, depending on the request specificity

**Review by pbernad-leadin — CHANGES_REQUESTED** [review]

File: src/HttpHelper/HttpClientHelper/HttpClient.php (hunk 1 above)
Comment (rtwent) [reply src/HttpHelper/HttpClientHelper/HttpClient.php:133]: @leadinpier @pbernad-leadin 
Guys. Whose comment is more relevant?
https://lead-in.monday.com/boards/72721864/pulses/5154589834/posts/3560280413
or
https://lead-in.monday.com/boards/72721864/pulses/5154589834/posts/3566628920

Comment (leadinpier) [reply src/HttpHelper/HttpClientHelper/HttpClient.php:133]: > @leadinpier @pbernad-leadin Guys. Whose comment is more relevant? https://lead-in.monday.com/boards/72721864/pulses/5154589834/posts/3560280413 or https://lead-in.monday.com/boards/72721864/pulses/5154589834/posts/3566628920

@rtwent my comment belonged to 
https://lead-in.monday.com/boards/72721864/pulses/4680529671
and not to
https://lead-in.monday.com/boards/72721864/pulses/5154589834

Comment (rtwent) [reply src/HttpHelper/HttpClientHelper/HttpClient.php]: Done

**Review by pbernad-leadin — APPROVED** [review]

**Pull request merged**